examples/kafka_spark_demo/feature_repo/features.py

```diff
@@ -0,0 +1,50 @@
+from datetime import timedelta
+from pyspark.sql import DataFrame
+
+from feast import (
+    FeatureView,
+    Field,
+)
+from feast.stream_feature_view import stream_feature_view
+from feast.types import Float32, Int32
+
+from data_sources import *
+from entities import *
+
+customer_stats_view = FeatureView(
+    name="customer_stats",
+    description="customer features",
+    entities=[customer],
+    ttl=timedelta(seconds=8640000000),
+    schema=[
+        Field(name="current_balance", dtype=Float32),
+        Field(name="avg_passenger_count", dtype=Float32),
+        Field(name="lifetime_trip_count", dtype=Int32),
+    ],
+    online=True,
+    source=customer_stats_batch_source,
+    tags={"production": "True"},
+    owner="test1@gmail.com",
+)
+
+@stream_feature_view(
+    entities=[driver],
+    ttl=timedelta(seconds=8640000000),
+    mode="spark",
```

Comment:
nit maybe the mode should be `pyspark`

Reply:
agreed, but this requires a change to `contrib` so I'll do this separately after the tutorial lands